### PR TITLE
Add Genesis token

### PR DESCRIPTION
**Pull request by Thisuu**

### CODE REVIEW

**Review by thedarkjester — CHANGES_REQUESTED** [review]
Please could you verify the contract as mentioned in the main README
```
IMPORTANT: Before adding a new token, it is mandatory to verify the token's smart contract. This ensures the authenticity and security of the token. Contract verification should be done through [LineaScan's](https://lineascan.build/) or [Etherscan's](https://etherscan.io/) contract verification tools.
```

**Comment by Thisuu** [comment]
> Please could you verify the contract as mentioned in the main README
> 
> ```
> IMPORTANT: Before adding a new token, it is mandatory to verify the token's smart contract. This ensures the authenticity and security of the token. Contract verification should be done through [LineaScan's](https://lineascan.build/) or [Etherscan's](https://etherscan.io/) contract verification tools.
> ```

Thanks for the heads up! Contract is now verified.

File: json/linea-mainnet-token-shortlist.json [hunk 1]
```diff
@@ -223,6 +223,21 @@
...
     },
+    {
+      "chainId": 59144,
+      "chainURI": "https://lineascan.build/block/0",
+      "tokenId": "https://lineascan.build/address/0x42d4Eb291C00a243C7Cbc2759B47892ED1852a9d",
+      "tokenType": [
+        "native"
+      ],
+      "address": "0x42d4Eb291C00a243C7Cbc2759B47892ED1852a9d",
+      "name": "GENESIS",
```
Comment (thedarkjester) [inline json/linea-mainnet-token-shortlist.json:234]: Please could you correct the casing on the name
image

Comment (Thisuu) [reply json/linea-mainnet-token-shortlist.json]: My bad, Fixed!

**Review by thedarkjester — APPROVED** [review]

**Pull request merged**